jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
+  preset: 'ts-jest',
+  coverageDirectory: '__coverage__',
+  collectCoverage: true,
+  transformIgnorePatterns: [
+    'node_modules/(?!@patternfly|lodash-es|@console|@novnc|@spice-project|@popperjs|i18next)'
+  ],
+  coverageReporters: [
+    'json',
+    'lcov',
+    'text',
+    'text-summary'
+  ],
+  collectCoverageFrom: [
+    'packages/*/src/**/*.{js,jsx,ts,tsx}',
+    '!**/node_modules/**',
+    '!**/dist/**'
+  ],
+  testURL: 'http://localhost',
+  testRegex: '.*\\.(spec|test)\\.(ts|tsx|js|jsx)$',
+  testEnvironment: 'jsdom',
+  testPathIgnorePatterns: [
+    '<rootDir>/node_modules/'
+  ],
+  roots: ['<rootDir>/packages/'],
+  globalSetup: '<rootDir>/config/globalSetup.ts',
+  setupFilesAfterEnv: ['<rootDir>/config/setupTests.ts'],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|css|less|scss)$': 'identity-obj-proxy',
```

Comment:
I guess this can also be shortened a bit.. `mp3` or `wav` etc. shouldn't be needed :smile:

Reply:
Yeah, many of these are not required, but I'd rather keep them here otherwise it could bite us in future.